platform-api/tests/test_content.py

```
@@ -0,0 +1,95 @@
+        return []
+
+# 2. Create a Fake Settings Provider
+def mock_get_settings():
+    mock = MagicMock(spec=WorkerSettings)
+    mock.required_secrets = {"admin_api_key": "test-admin-key"}
+    mock.optional_secrets = {"sanity_api_write_token": "test-write-token"}
+    mock.env_vars = {"SANITY_PROJECT_ID": "test"}
+    return mock
+
+# 3. Apply the overrides
+app.dependency_overrides[get_sanity] = lambda: MockSanityClient()
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)